src/components/pages/tutorials/Tutorials.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Icon from '../../layout/Icon/Icon'
+
+const Tutorials = () => (
+  <main className="tutorials">
+    <Icon
+      alt="calendrier"
+      className="calendar"
+      svg="calendar"
+    />
+    <p className="first-tutorial-text">
+      <span>
+        {"À partir d'aujourd'hui, tu as "}
+      </span>
+      <span className="text-highlight text-2-ans">
+        {'2'}
+        &nbsp;
+        {'ans'}
+      </span>
+      <span>
+        {' et '}
+      </span>
+      <span className="text-highlight text-500-euros">
+        {'500€'}
```

Comment:
```suggestion
        {`500${NON_BREAKING_SPACE}€`}
```
Avec `const NON_BREAKING_SPACE = '\u00A0'`
Il existe déjà dans le code, ça serait cool qu'il soit qu'à un seul endroit <3

https://twitter.com/typochef/status/962987516952539136